Package.swift

```diff
@@ -27,7 +27,7 @@ let strictConcurrencySettings: [SwiftSetting] = {
     if strictConcurrencyDevelopment {
         // -warnings-as-errors here is a workaround so that IDE-based development can
         // get tripped up on -require-explicit-sendable.
-        initialSettings.append(.unsafeFlags(["-require-explicit-sendable", "-warnings-as-errors"]))
+        initialSettings.append(.unsafeFlags(["-Xfrontend", "-require-explicit-sendable", "-warnings-as-errors"]))
```

Comment:
Just checking you enabled `strictConcurrencyDevelopment` locally and built the package after this change?

Reply:
I did, yeah